src/components/FeedbackButton.tsx

```diff
@@ -0,0 +1,105 @@
+import { EditIcon } from "@chakra-ui/icons";
+
+import {
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalFooter,
+    ModalBody,
+    ModalCloseButton,
+    Button,
+    useDisclosure,
+    IconButton,
+    Text,
+    Textarea
+} from "@chakra-ui/react";
+
+import React from "react";
+
+import { submitFeedback } from "../services/feedbackService";
+
+export function FeedbackButton() {
+    const { isOpen, onOpen, onClose } = useDisclosure();
+    const [feedbackString, setFeedbackString] = React.useState<string>("");
+    const [isFeedbackSubmit, setIsFeedbackSubmit] = React.useState<boolean>(false);
+
+    const handleInputChange = (e: any) => {
+        const inputValue = e.target.value;
+        setFeedbackString(inputValue);
+    };
+
+    const submitForm = async () => {
+        await submitFeedback(feedbackString);
+
+        // Communicate that feedback is submit
+        setIsFeedbackSubmit(true);
+
+        // Clear feedback and close
+        setFeedbackString("");
+    };
+
+    const customOnClose = () => {
+        onClose();
+        setIsFeedbackSubmit(false);
+    };
+
+    const modalContent = isFeedbackSubmit ? (
+        <ModalContent>
+            <ModalHeader>Thanks for your feedback!</ModalHeader>
+            <ModalCloseButton />
+            <ModalFooter>
+                <Button colorScheme="red" mr={1} onClick={customOnClose}>
```

Comment:
please avoid using non-standard colors that don't come from the theme. if you want an error color, we can add it to the theme. ("index.ts in acorn theme")